src/main/kotlin/app/revanced/patches/grindr/annotations/GrindrPatchCompatibility.kt

```diff
@@ -0,0 +1,8 @@
+package app.revanced.patches.grindr.annotations
+
+import app.revanced.patcher.annotation.Compatibility
+import app.revanced.patcher.annotation.Package
+
+@Compatibility([Package("com.grindrapp.android", arrayOf("9.15.0"))])
```

Comment:
If all fingerprints are done properly, the version constraint should not be necessary.